modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -101,23 +98,32 @@ You can change the details and settings of your autoinstallation profiles by nav
 Alternatively, navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
 
 
+
 [[autoyast]]
 == AutoYast Profiles
 
 An {ay} profile consists of a [guimenu]``Label`` that identifies the profile, an [guimenu]``Autoinstall Tree`` that points to an autoinstallable distribution, various options, and, most importantly, an {ay} installation file.
 
-The {ay} installation file is an XML file that give directions to the {ay} installer. {ay} calls it a "control file."
+The {ay} installation file is an XML file that give directions to the {ay} installer.
+{ay} calls it a "control file."
 For the full syntax of {ay} installation files, see https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
 
 {suse} provides templates of {ay} installation files that you can use as a starting point for your own custom files.
-You will find them at https://github.com/SUSE/manager-build-profiles in the directory named [path]``AutoYast``.
+You will find them at https://github.com/SUSE/manager-build-profiles in the [path]``AutoYast`` directory.
 Each of these profiles requires you to set some variables before you use it.
 Check the [path]``README`` file included with the script to determine which variables you need.
 For more information about using variables in {ay} scripts, see xref:client-configuration:autoinst-profiles#variables[Variables].
 
-Here are the most important sections in the {ay} installation file for installing with SUSE Manager:
+These are the most important sections in the {ay} installation file for installing with {productname}:
 
 * ``<add-on>`` section allows to add child channels to the installation
++
+  See https://doc.opensuse.org/projects/autoyast/#Software-Selections-additional with an ``<add-on>`` example
+
+* ``<software>`` section allows to specify ``<product>`` for the Unified Installer
```

Comment:
I cannot add the comment at the right place, but "software" exists again below. I think we need it only 1 time.

Reply:
thanks for spotting!